plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -1048,6 +1115,27 @@ yargs(process.argv.slice(2))
       process.exitCode = extract(argv.pluginPackages, argv.outputPlugins);
     }
   )
+  .command(
+    'pack <pluginPath> <outputDir>',
```

Comment:
Can you please add docs to these spots?
- headlamp-plugin/README.md
- headlamp/docs/development/plugins/building.md

Reply:
I will add to README.md but for the building, we need more docs, as pack is tied to install as well.